src/LibraryWindow.vala

```diff
@@ -436,7 +436,7 @@ public class Noise.LibraryWindow : LibraryWindowInterface, Gtk.Window {
         saved_state_settings = new GLib.Settings ("org.pantheon.noise.saved-state");
 
         set_default_size (saved_state_settings.get_int ("window-width"), saved_state_settings.get_int ("window-height"));
-
+        move (saved_state_settings.get_int ("window-x"), saved_state_settings.get_int ("window-y"));
```

Comment:
You should check that the values are not `-1, -1`. Otherwise the window will be placed at `0, 0` instead of the window manager default (center) :)